packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimePickerContent.tsx

```diff
@@ -216,16 +216,17 @@ const FullScreenForm = (props: FormProps) => {
 const EmptyRecentList = memo(() => {
   const theme = useTheme2();
   const styles = getEmptyListStyles(theme);
+  const emptyRecentListText = t(
+    'time-picker.content.empty-recent-list-text1',
```

Comment:
That's an interesting one. Not sure what's the best way to have a single entry. Should we use [nesting](https://www.i18next.com/translation-function/nesting) here? Or if we go with two separate keys we should probably have more meaningful names, not number suffixes. @JoaoSilvaGrafana what do you think?

Reply:
I think the simplest would be to have more meaningful names for them? How about `time-picker.content.empty-description` and `time-picker.content.empty-read-more`?

Reply:
@grafana/grafana-frontend-platform wdyt about this one?